app/views/comments/_comment_form.html.erb

```diff
@@ -54,7 +54,7 @@
                           commentable_creator: commentable.by_anonymous_creator? ? t(".anonymous_creator") : get_commenter_pseud_or_name(commentable),
                           commentable_link: commentable_description_link(commentable)) %></span>
             <% end %>
-            (<%= allowed_html_instructions %>)
+            <%= allowed_html_instructions(strip_images: comment.hidden_by_admin || Comment.commentable_object(commentable).class.name.in?(ArchiveConfig.PARENTS_WITH_IMAGE_SAFETY_MODE)) %>
```

Comment:
Is it possible to use `use_image_safety_mode?` from the comment model here instead?

Reply:
Not as it stands. `use_image_safety_mode?` checks through `Comment.parent` and `Comment.pseud`, which aren't set until the comment is saved. I considered rewriting it, but that would require something like
```ruby
def use_image_safety_mode?(commentable = nil)
  (persisted? && pseud_id.nil?) || hidden_by_admin || (persisted? ? parent_type : Comment.commentable_object(commentable)).in?(ArchiveConfig.PARENTS_WITH_IMAGE_SAFETY_MODE)
end
  ```
  so since this is _probably_ the only place that will have non-persisted comments, I figured it'd be cleaner just to leave it as-is. Should it be made into it's own function instead?